### PR TITLE
Verify super sale 10130

### PR DESCRIPTION
**Pull request by kammola**

### CODE REVIEW

File: prusti-tests/tests/verify_overflow/pass/competitive_programming/dp/SuperSale_10130.rs [hunk 1]
```diff
@@ -0,0 +1,268 @@
+// compile-flags: -Zdisable_more_complete_exhale
+// https://onlinejudge.org/external/101/10130.pdf
+use prusti_contracts::*;
+
+use prusti_contracts::*;
+
```
Comment (Aurel300) [inline prusti-tests/tests/verify_overflow/pass/competitive_programming/dp/SuperSale_10130.rs]:
```suggestion
```

Comment (kammola) [reply prusti-tests/tests/verify_overflow/pass/competitive_programming/dp/SuperSale_10130.rs]: I don't understand why should it be removed. Removing it gives errors in the editor.

Comment (Aurel300) [reply prusti-tests/tests/verify_overflow/pass/competitive_programming/dp/SuperSale_10130.rs]: The `use` is there twice, my suggestion was just to remove the duplicate.

Comment (kammola) [reply prusti-tests/tests/verify_overflow/pass/competitive_programming/dp/SuperSale_10130.rs]: Done.

**Comment by vakaras** [comment]
Superseded by #512

**Pull request closed without merging**